source/index.txt

```diff
@@ -13,10 +13,19 @@ Introduction
 ------------
 
 Welcome to the documentation site for the official {+provider-long+}.
-You can add the provider to your .NET application as an object-relational mapper (ORM)
-to work with data in MongoDB.
+
+You can add the {+provider-short+} to your .NET application as an object-relational mapper
```

Comment:
Suggestion:

I think it would be good to use source constant for all instances of a value to enforce consistency in capitalization. No need for proactive changes everywhere, but would be helpful to use it in the same PR as it's introduced.

```suggestion
You can add the {+provider-short+} to your {+language-dotnet+} application as an object-relational mapper
```